src/libraries/Common/tests/System/Xml/BaseLibManaged/BaseLibManaged.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace>WebData.BaseLib</RootNamespace>
-    <Configurations>netstandard2.0-Debug;netstandard2.0-Release</Configurations>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Same question here, what will the convention be? Should we use `TargetFrameworks` when there is only one framework for the project or should we use `TargetFrameworks`?

Reply:
Currently we cant use TargetFramework even for the single TargetFramework because we derive some properties from TargetFramework in our Directory.Build.props and use them in .csproj to include files eg TargetWindows.
So we depend on outer build to set this as a global property

Reply:
@Anipik and I discussed this and we could hack it to make it work, but for now it's easier to require TargetFrameworks. 

Essentially the fact that we derive a lot of things from TargetFramework then use those derived properties in the project body (TargetsNetstandard, TargetsWindows, etc) we cannot derive those properties from something in the project body.  The way this worked before was we had configuration.props insert itself in the middle of the property evaluation early enough to set this before deriving other properties.  The extra file is a nuisance, so now we just use a property in the project and require an outer build.

A hack to support TargetFramework would be to force an outer build in this case, but its easier to just to use TargetFrameworks across the board.

Reply:
Makes sense, thanks for the detailed explanation.

Reply:
We should make sure to document that somewhere.